config/tests/samples/create/samples.go

```diff
@@ -15,6 +15,7 @@
 package create
 
 import (
+	goerrors "errors"
```

Comment:
Nit (No need to fix), but I normally prefer to import apimachinery's errors as apierrors